domains/tuanpham.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Tun52299",
+           "email": "",
+           "discord": "699291160157618226",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.NO4Prmw4__zqB_JFqETf5p4zidLyqmBWvdQdtXTZ1AkyrCUUXRzqJX_Ls4rVWgkO_L-7tJgwLqo1Kd_bLWXm9KgI28TiqxqtAyuHBvQ2k_Ljy5AKzv1e7f_0SsJYGdONyz8JcRwE7lnzm9fkjbGwtLmmbY79vX4z0WckcIQXJZXsToT2cp33M_11wb1F9JpTDBq4SKvBcu11jeTSv3J_gyeL8JMFQQ87F16U30QGY98mtMJq7x0LWWqS2FevqPRyuiFwPOVuiir0qLVfCN9rOj3hnjsQh5ZDXaLCW-KoIBxJtAouQx7gOWsvAgEw9QUH4Z8NRKuklsIpk1ZbSSRgHw.VdDmy0RUUAIVXJfbm8FO3Q.nxTc-dx3PdfwHgGdk3yXfbsJj2Kdr6UrWSvlF9_yecVQH1eOETVxL6iwBdiNdLxfKKrP76gTS-9GK6j-cL1Ix-W5CaeyX4l2Jz_0nDgRVE8.zGKqhWqeyusy1PwiiMxnJA"
+        },
+    
+        "record": {
+            "CNAME": "tun52299.github.io"
```

Comment:
Please give a preview of your site, since it redirects to is-a.dev. (I assume you tried setting it up)

Reply:
This is a preview of my website https://github.com/Tun52299/Tun52299.github.io